blueprints/app/files/bower.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
The user will not be presented with any message!? We automatically create the file if we have to, though not through bower init.